Source/EasyNetQ/SubscriptionConfiguration.cs

```diff
@@ -217,25 +212,25 @@ public ISubscriptionConfiguration WithQueueName(string queueName)
 
     public ISubscriptionConfiguration WithMaxLength(int maxLength)
```

Comment:
do we need `Queue` prefix for configuration methods like this?
`WithMaxPriority` -> `WithQueueMaxPriority`

Reply:
> do we need `Queue` prefix for configuration methods like this?
> `WithMaxPriority` -> `WithQueueMaxPriority`

Nope. It breaks a lot of existing code.